pkg/sql/conn_executor.go

```diff
@@ -860,7 +877,7 @@ func (ex *connExecutor) run(ctx context.Context) error {
 			}
 			return err
 		}
-		log.VEventf(ex.Ctx(), 2, "[%s pos:%d] connEx executing cmd %s",
+		ex.sessionEventf(ex.Ctx(), "[%s pos:%d] connEx executing cmd %s",
```

Comment:
since you're diving back into this area, can I further attempt to convince you to make this message less verbose? There are usually many of them right next to each other, so having `connEx executing cmd` is pretty redundant IMO.

What about `["%s pos:%d] pgwire %s` or something like that?